docs/cli.md

```diff
@@ -0,0 +1,104 @@
+# CLI
+
+Operates on directories as a unit of work:
+
+```
+my-model/
+  substratus.yaml <---- Reads Dataset/Model/Server/Notebook from here.
+  Dockerfile <--- Optional
+  run.ipynb
+```
+
+## Alternative Names
+
+"AI CLI":
+
+```bash
+# Looks for "ai.yaml"...
+
+ai notebook .
+ai nb .
+ai run .
+```
+
+## Notebook
+
+```bash
+strat nb (notebook)
+```
+
+## List
+
+```bash
+strat ls (list)
```

Comment:
+1 to `strat`, -1 to `ai`

Reply:
I think `sub` has my preference right now

Reply:
sub it is!